services-marionette/firefox_services_tests/runtests.py

```diff
@@ -0,0 +1,17 @@
+from firefox_puppeteer.testcases.base import FirefoxTestCase
```

Comment:
We export this class via __init__.py. So no need to specify base here. You can do it via `from firefox_puppeteer.testcases import FirefoxTestCase`.